packages/next/client/image.tsx

```diff
@@ -232,6 +249,14 @@ export default function Image({
     }
   }
 
+  if (!layout) {
+    if (sizes) {
+      layout = 'responsive'
+    } else {
+      layout = 'fixed'
```

Comment:
To keep compatibility with Next 10 as it was released let's go back to intrinsic, sorry for the flip-flop here!
```suggestion
      layout = 'intrinsic'
```